extensions/wikia/TrackingOptIn/src/vendorlist.json

```diff
@@ -1362,23 +1362,6 @@
 			"legIntPurposeIds": [],
 			"featureIds": []
 		},
-		{
-			"id": 97,
-			"name": "LiveRamp, Inc.",
-			"policyUrl": "www.liveramp.com/service-privacy-policy/",
-			"purposeIds": [
-				1,
-				2,
-				3,
-				4,
-				5
-			],
-			"legIntPurposeIds": [],
-			"featureIds": [
-				1,
-				2
-			]
-		},
```

Comment:
I'm no sure if this vendor list is still being used. I think it's some kind of leftover. @dmnsobczak can you confirm it? Anyway, LiveRamp should stay on the list since it's the list of all vendors.

Reply:
That's a leftover, not used. You can remove whole file to be honest.

Reply:
OK, thanks! Done.